### ISSUE TITLE
可以提供一下百度网盘的提取码吗，过期了

### ISSUE THREAD

**Issue by niehao0209** [huangjin520/EMGANet#9]
谢谢

**Comment by huangjin520** [comment]
我刚刚试了下可以登录提取到文件，会不会是浏览器的原因，要不咱们换一个浏览器再试试

**Comment by huangjin520** [comment]
<img width="1295" height="608" alt="Image" src="https://github.com/user-attachments/assets/f9ab7392-c066-42b1-9975-287d23133ce3" />

————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
<img width="911" height="481" alt="Image" src="https://github.com/user-attachments/assets/5cf0b6bd-3a00-45c3-bf43-2da273c1d32a" />

**Comment by niehao0209** [comment]
非常感谢作者能回邮箱😭
我说错了，是提取码是错误的，我输入你提供的，提示提取码错误，打扰了，希望作者大大能够提供一下正确的提取码

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2026年1月8日(周四) 下午5:36
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [huangjin520/EMGANet] 可以提供一下百度网盘的提取码吗，过期了 (Issue #9)


huangjin520 left a comment (huangjin520/EMGANet#9)
 
我刚刚试了下可以登录提取到文件，会不会是浏览器的原因，要不咱们换一个浏览器再试试

**Comment by huangjin520** [comment]
通过网盘分享的文件：EMGANet
链接: https://pan.baidu.com/s/140BzQjkAwXHqa9zABXr_qg?pwd=luck 提取码: luck

我重新生成了一个链接，试试看

**Comment by niehao0209** [comment]
非常感谢作者大大，祝你生活顺利，我是一名计算机科学与技术的研一学生，老师的研究方向就是医学图像分割，非常感谢能得到你的帮助，可能在后续学习你的这篇优秀论文的过程中会出现一些疑问，可能需要请教您一下，如有打扰，请原谅，非常感谢




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "huangjin520/EMGANet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2026年1月8日(星期四) 晚上6:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [huangjin520/EMGANet] 可以提供一下百度网盘的提取码吗，过期了 (Issue #9)



huangjin520 left a comment (huangjin520/EMGANet#9)
 
通过网盘分享的文件：EMGANet
 链接: https://pan.baidu.com/s/140BzQjkAwXHqa9zABXr_qg?pwd=luck 提取码: luck
 
我重新生成了一个链接，试试看

**Comment by huangjin520** [comment]
祝你一切顺利哦，顺利发出自己的文章！！！！
顺便说一下，我也是学生，一起加油！

**Comment by niehao0209** [comment]
作者大大，现在我在做相关的对比实验，但是现在我一直有一个疑惑，对比实验那块很多别人的模型不是在超声数据集上进行的，但那些比较的数据结果是怎么来的呢，而且有一些没有源码的话我该怎么去进行对比呢，这个问题一直困惑我很久啦，非常抱歉打扰作者，但是我真的很需要你帮我解答一下，感谢作者大大

**Comment by huangjin520** [comment]
不好意思，消息没及时收到。关于你的问题，我分以下几个部分回答：

# 🚩 1. 没有源码

**如果某个对比算法没有公开源码，建议直接更换为其他具有公开实现的对比算法。**

# 🚩 2. 对比实验的正确做法

对于论文中的对比实验，不能直接引用原论文中的指标和图片，而应该：

## ✅ 重新训练对比算法

并确保与本文方法采用完全一致的实验配置，包括：

* **Dataset Split（数据集划分）**
* **Evaluation Metrics（评价指标）**
* **Batch Size**
* **Loss Function**
* **Training Epochs**
* **Software Environment（软件环境）**
* **Hardware Environment（硬件环境）**

> **背后的原理是控制变量法**
> 只有在所有实验条件保持一致的情况下，才能保证实验结果的公平性和可信度。
> 因此，对比算法应在与本文方法完全相同的配置下重新训练和测试。若在相同条件下其性能仍低于本文方法，则可以证明性能提升来源于算法本身，而非实验设置差异，从而更有力地支撑本文方法的有效性和先进性（SOTA）。

感谢你的关注，希望这些说明能够帮助你，祝你论文顺利accept!
